app/src/routes/v1.js

```diff
@@ -32,6 +31,6 @@ router.get('/api-spec.yaml', (_req, res) => {
 router.use('/checks', keycloak.protect(), checksRouter);
 
 /** Email Router */
-router.use('/email', keycloak.protect(`${config.get('keycloak.clientId')}:EMAILER`), emailRouter);
```

Comment:
Intentional to take away the EMAILER role check here? Or leftover from testing?

Reply:
This needs at bare minimum `keycloak.protect()`, otherwise anyone is allowed to invoke the routes.

Reply:
yup, lost in testing.  we need that! back in next commit